pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -46,38 +47,24 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 			},
 			ServiceAccountName: serviceAccountName,
 			RestartPolicy:      v1.RestartPolicyAlways,
-			Volumes:            opspec.DaemonVolumes(mgrConfig.DataPathMap, mgrConfig.ResourceName),
-			HostNetwork:        c.HostNetwork,
+			Volumes: append(
+				opspec.DaemonVolumes(mgrConfig.DataPathMap, mgrConfig.ResourceName),
+				keyring.Volume().Admin(), // ceph config set commands want admin keyring
```

Comment:
We don't need this anymore if not using init containers, right?

Reply:
Oh wait, the two existing init containers still need it and the keyring volume just moved from below to here, right?

Reply:
The extra admin volume is only needed when the init containers are setup. I forgot to revert this when I had originally written the solution to use init containers as well, but then moved to a solution where we issue the configuration commands directly in the operator. Fixed now.